src/app/_components/HaveAnAccount/HaveAnAccount.tsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import { Box, Button, Typography, styled } from "@mui/material";
+
+{/*Using styled-components approach to apply styles to the Typography component*/}
+const GrayText = styled(Typography)({
+  color: '#B0B0B0', // Light gray color
+  cursor: 'pointer', // Change cursor to pointer to indicate it's clickable
+  '&:hover': {
+    color: '#9A9A9A', // Darker gray color on hover to indicate interactivity
+  },
+});
```

Comment:
Cool to see you use styled components, ultimately when we have the theme setup correctly, all these should happen automatically.